geonode_mapstore_client/static/mapstore/configs/localConfig.json

```diff
@@ -21,7 +21,7 @@
         }
     ],
     "geoNodeApi": {
-        "endpointAdapter": "/mapstore/rest",
+        "newEndpointAdapter": "/api/v2/maps/",
```

Comment:
I think we should not rename the `endpointAdapter` but instead move all the new requests in the api/geonode/v2/index.js file, use directly the exiting endpointV2  and rename the request as follow:
- creatMapStoreMap -> createMap
- getMapStoreMapById -> getMapByPk
- updateMapStoreMap -> updateMap

This will be similar to the current GeoStory workflow